packages/runtime-dom/src/directives/vModel.ts

```diff
@@ -84,7 +84,7 @@ export const vModelCheckbox: Directive<HTMLInputElement> = {
         if (i > -1) {
           assign([...modelValue.slice(0, i), ...modelValue.slice(i + 1)])
         } else {
-          assign(modelValue.concat(elementValue))
+          assign(modelValue.concat([elementValue]))
```

Comment:
why you prefer creating an array?
The previous version is correct.

[concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat)

Reply:
yes... you are right.
the code of vue2.x misled me. 😂